include/nodes/internal/FlowView.hpp

```diff
@@ -27,6 +28,8 @@ class NODE_EDITOR_PUBLIC FlowView
 
   void setScene(FlowScene *scene);
 
+  void setFlowViewStyle(const FlowViewStyle* style);
```

Comment:
Who is the owner of this pointer and who must remove it?
I'd simply pass a plain const reference. It's cheap to copy three colors.

Reply:
I changed my approach to expose StyleCollection::setFlowViewStyle in the public interface. Please re-review.